tests-schema/ModelTest.php

```diff
@@ -134,7 +127,7 @@ protected function init(): void
         parent::init();
 
         $this->addField('name');
-        $this->addField('password', ['type' => 'password']);
```

Comment:
don't we support field type "password" anymore?

Reply:
I commented it out, so we can get to it later once we switch to DBAL types